notebooks/foundations/energy-balance-model.ipynb

```diff
@@ -34,37 +26,27 @@
    "metadata": {},
    "source": [
     "## Overview\n",
-    "If you have an introductory paragraph, lead with it here! Keep it short and tied to your material, then be sure to continue into the required list of topics below,\n",
     "\n",
-    "1. This is a numbered list of the specific topics\n",
-    "1. These should map approximately to your main sections of content\n",
-    "1. Or each second-level, `##`, header in your notebook\n",
-    "1. Keep the size and scope of your notebook in check\n",
-    "1. And be sure to let the reader know up front the important concepts they'll be leaving with"
+    "This tutorial focuses on introducing the fundamental concepts of the energy balance model and radiative feedback, and establish notations and definitions for the rest of the cookbook. \n",
```

Comment:
"This tutorial focuses on introducing the fundamental concepts of the energy balance model and radiative feedback, as well as establishing notations and definitions that will be used in future sections. \n",